mlflow/utils/environment.py

```diff
@@ -453,18 +454,27 @@ def _is_mlflow_requirement(requirement_string):
             )
 
 
-def _generate_mlflow_version_pinning():
+def _generate_mlflow_version_pinning() -> str:
     """
     Determines the current MLflow version that is installed and adds a pinned boundary version range
```

Comment:
nit: We always specifies a particular version, so no longer range?

Reply:
Thanks for the catch, will fix!